_layouts/register.html

```diff
@@ -253,7 +253,6 @@
       </div>
     </div>
   </div>
-
 </body>
 {% if site.google_analytics %}
```

Comment:
This looks the issue you tried to fix? If the “if” GA block does what it seems to do, move it inside the body (but the whole block, including the “endif”).

(May affect more files)